.github/workflows/ci.yaml

```diff
@@ -1,13 +1,58 @@
-name: release
+name: CI
+
 on:
   push:
-    tags:
-      - "v*"
+  pull_request:
+
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: ["1.15.x", "1.16.x"]
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+
+      - name: Run test
+        run: make test COVERAGE_DIR=/tmp/coverage
+
+      - name: Send goveralls coverage
+        uses: shogo82148/actions-goveralls@v1
+        with:
+          path-to-profile: /tmp/coverage/combined.txt
+          flag-name: Go-${{ matrix.go }}
+          parallel: true
+
+  test-finish:
```

Comment:
~Rename to `send-coverage` and add a human-readable `name: Send coverage`~

EDIT: This looks like a duplicate sending of coverage?

Reply:
Oh, NVM, it looks like this notifies github that coverage has pass. Let's name this `check-coverage` with a human readable `name: Check coverage`